src/style.css

```diff
@@ -0,0 +1,82 @@
+.header{
+    background-color: white;
+    display: flex;
+    justify-content: right;
+    height: 70px;
+    padding: 20px;
+}
+
+.header ul li a{
+   font-size: 26px;
+   color: black;
+   text-decoration: none;
+   color: blue;
+
+}
+
+header{
+    border: 2px solid lightgray;
+}
+
+.main-container{
+    width: 100%;
+    display: flex;
+}
+.personal-infomation{
+    width: 100%;
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+/*about section */
+.about-section{
+    width: 95%;
+    background-color: white;
+    padding: 20px;
+    border: 2px solid lightgray;
+    margin-bottom: 5px;  
+}
+/*experience section*/
+.experience-section{
+    width: 95%;
+    background-color: white;
+    padding: 20px;
+    height: 400px;
+    border: 2px solid lightgray;
+}
+
+/*sidebar*/
+.sidebar{
+    width: 30%;
+    background-color: white;
+    padding: 20px;
+    border: 2px solid lightgray;
+    margin-top: 5px;
+    margin-bottom: 5px;
+
+
+}
+
+/*avatar section*/
+.avatar{
+    margin-top: 50px;
+}
+
+.avatar img { 
+    border-radius: 50%;
+    object-fit: cover;
+}
+/*footer section  */
+.FooterSection{
+        background-color: white;
+        height: 70px;
+        padding: 20px;
+        display: flex;
+        justify-content: left;
+        border: 2px solid lightgray;
+    }
```

Comment:
keep the naming convention consistent for all your css classes.
Above you have kebab-case (.about-section), here you have PascalCase ( .FooterSection ).
its better to use the same case everywhere, kebab-case or camelCase is most commonly used with css. use one of these.

Reply:
fixed!